databases/mysql8/Portfile

```diff
@@ -69,6 +70,7 @@ if {$subport eq $name} {
 
     use_parallel_build  yes
 
+    # Will require C++17 as of 8.0.27
```

Comment:
https://dev.mysql.com/doc/refman/8.0/en/source-installation-prerequisites.html says:
> As of MySQL 8.0.27, MySQL 8.0 source code permits use of C++17 features.

so I have reminders to update `compiler.cxx_standard` to `2017` when updating to 8.0.27.